ui/src/overview.tsx

```diff
@@ -321,6 +323,9 @@ function ParticipantsGroupSection(props: {
       }}
     >
       <GridLayout rows height="auto">
+        {!!props.groupSection.name && (
```

Comment:
Use the `sectionName` helper in `cohort.ts` and always display the name. Also move it into the grey above the dropdowns and horizontally align it with them.

Reply:
Nice, didn't see `sectionName` first time around. I've updated the screenshot in the description after addressing the comments.